src/components/TodoItem.vue

```diff
@@ -40,6 +42,10 @@
         @edit="onEdit"
         @cancel="onCancel"
       />
+
+      <div v-if="isDone" class="todo__footer-completedIn">
+        Completed in {{ todo.doneIn }} {{ todo.doneIn === 0 ? "day" : "days" }}
```

Comment:
Always move logics to method or computed property to make MARKUP clean as markup easily gets messy.